.travis.yml

```diff
@@ -1,23 +1,41 @@
 branches:
   except:
-  - donttestme
+    - donttestme
+
 language: bash
+
 sudo: required
+
 env:
   global:
     - HADOLINT_VERSION=1.17.1
+    - SHELLCHECK_VERSION=latest
```

Comment:
Generally speaking you always want to perform tests with the same version. This is done to not have an external component influence testing results (breakage or false positives through changed testing dependencies).

Reply:
Good point. I will change this.

Reply:
This was just corrected with 3233034, but 286b53c did the trick due to a wrong `cp` command